v20.1/inverted-indexes.md

```diff
@@ -1,17 +1,21 @@
 ---
 title: Inverted Indexes
-summary: Inverted indexes improve your database's performance and usefulness by helping SQL locate schemaless data in a JSONB column.
+summary: Inverted indexes improve your database's performance and usefulness by helping SQL locate data in JSONB or ARRAY columns.
 toc: true
 ---
 
-Inverted indexes improve your database's performance by helping SQL locate the schemaless data in a [`JSONB`](jsonb.html) column.
+Inverted indexes store mappings from values within a column (such as a [`JSONB`](jsonb.html) document) to the row that holds that value.They are used to speed up containment searches, e.g., "show me all of the rows from this table which have a JSON column that contains the key-value pair `{"location":"NYC"}`". Inverted indexes are commonly used in [document retrieval systems](https://en.wikipedia.org/wiki/Document_retrieval).
```

Comment:
This is much better. I still think we need an adjective on `column` to show that it's not all columns. These columns contain tokens, or items. Container datatypes? Composite datatypes?

nit: `value.They` needs a space in between

Reply:
Makes sense - I went with "container column" since it seems simplest of the available options - "composite" sounds a bit more complex and I'd rather not get too much into explaining these here in an overview (at least, for the scope of this change - I think this discussion points to a need to refactor this page to be more comprehensive at some point going forward)

Fixed the period spacing typo, thanks for that!